Basic Rules.txt

```diff
@@ -643,7 +643,7 @@ Instead of dealing full damage, a character can try to kick/shove a human sized
 opponent to force movement by rolling for a normal unarmed/armed melee attack 
 with an additional -2 miss penalty against the opponent's Guard DC. If the attack 
 succeeds, it yields 1/2 the damage it normally would and pushes the target 
-1m for every 5 strength you have.
+1m for up to every 5 strength you have.

```

Comment:
Can you add a pair of commas or parenthesis around "up to"? The prepositional phrase, or whatever that is, isn't 'essential to the meaning of the sentence,' and would probably be easier to read with additional help.

Reply:
The fix you want is to move the 'up to' before the '1m', so you have 'up to 1m per 5 strength'. Grammatically it cannot be turned into an appositive because it isn't a clause.

Reply:
Can do! :)